DentOS_Framework/DentOsTestbed/configuration/testbed_config/sit/testbed.json

```diff
@@ -126,6 +128,7 @@
             },
             "serialDev":"/dev/ttyUSB3",
             "baudrate": 115200,
+            "mediaMode":"fiber",
```

Comment:
Is it possible to use here "mixed"? AS4224 and TN48M have both fiber and copper.

Reply:
the devices where i specified it are 48P 10G fiber
for the ones that are 48P 1G + 4P 10G i did not specified anything.
+ as of now it is not used on DUT is used only on Keysight gear that supports on same port both modes.